_sections/act_and_prepare/en/04-myths/03-near-me.md

```diff
@@ -1,7 +1,9 @@
-{:.only_toc}
-### There are no infections near me.
+{:.myth}
```

Comment:
whatever we do here should be reconciled with the classes for "Things you should do" and "Things you should not do"